### ISSUE TITLE
Brave ad blocker issue

### ISSUE THREAD

**Issue by EnderDovahkin** [photopea/photopea#6841]
Your Operating System: windows 1o
Your Web Browser: Brave

problem is simple, brave has a built in adblock called brave shields, it can be turned off by choice of the user for specific sites. turning it off for photopea doesn't turn off adblock warnings.

**Comment by photopea** [comment]
Yeah, when I used Brave, it seems like it constantly keeps blocking ads, no matter what settings you use. I think it is a bug in Brave, you should report it to Brave.

@JoshuaKimsey  you use Brave, right?
